src/concrete/ml/sklearn/base.py

```diff
@@ -1766,6 +1768,11 @@ def __init__(self, n_bits: int = 3):
         self._y: numpy.ndarray
         # _q_fit_X_quantizer: The quantizer to use for quantizing the model's training set
         self._q_fit_X_quantizer: Optional[UniformQuantizer] = None
+        # Topk labels
+        self._topk_labels: numpy.ndarray
```

Comment:
I you keep the following attributes , don't forget to : 
- add typings
- add a small comment of the form `# variable: description`

Reply:
this variable has been removed